Jenkinsfile_CNP

```diff
@@ -101,10 +103,9 @@ withPipeline(type, product, component) {
     sh """
       eval \$(./bin/variables/load-preview-environment-variables.sh ${CHANGE_ID})
       ./bin/add-roles.sh
-      ./bin/pull-latest-release-asset.sh civil-ccd-definition civil-ccd-definition.zip
-      ./bin/pull-latest-release-asset.sh civil-camunda-bpmn-definition civil-camunda-bpmn-definition.zip
+      ./bin/pull-latest-camunda-files.sh ${camundaBranch}
+      ./bin/pull-latest-ccd-files-from-branch.sh ${ccdBranch}
```

Comment:
Can we harmonize the shell scripts names?

Like both ending with "-from-branch.sh" or neither.